Cargo.toml

```diff
@@ -13,6 +13,7 @@ default = []
 asm = []
 # Detect if requirements are met, and enable asm feature when we can.
 detect-asm = []
+chaos_mode = ["ckb-vm-definitions/chaos_mode"]
```

Comment:
Maybe rename this as `enable_chaos_mode_by_default`? It's longer but it tells what it really is.

Reply:
Agree, it makes it clearer